examples/text_labels_and_annotations/demo_annotation_box.py

```diff
@@ -22,27 +22,28 @@
 
 # Define a 1st position to annotate (display it with a marker)
 xy = (0.5, 0.7)
-ax.plot(xy[0], xy[1], ".r")
```

Comment:
There's a lot of quote changes. Is that intentional? Please note that we currently do not have a convention on single vs. double quotes. Both are acceptable, but we don't want to switch back and forth in existing code.

Reply:
Fair enough.  It was just to make sure there was one consistent type within this script, and I thought I had seen more single than double within Matplotlib.  Happy to change it back if that is not desired.

Reply:
Yes please. If we go for consistent string styling this will need a separate discussion and a decision which way to go (e.g. black enforces double quotes). Until now, let's keep existing quotes as they are.

Reply:
Done.